tests/test_server.py

```diff
@@ -181,6 +181,46 @@ async def test_autostart_batched(client, server, environment):
     assert len(sessionendpoint._sessions) == 1
 
 
+async def test_environment_setting_n_versions(client, server):
+    """
+    This test creates two environments with different values for the environment
+    setting AVAILABLE_VERSIONS_TO_KEEP and checks that the number of versions is
+    in line with the desired setting
+    """
+
+    env_1_expected_value = 30
```

Comment:
I would use lower number here, to make the testcase faster.

Reply:
It's quite long currently indeed (~7s)
I will override the default value of 100 as well for `env_2` after checking that it is set correctly